src/WebJobs.Extensions.DurableTask/Listener/DurableTaskListener.cs

```diff
@@ -51,21 +52,25 @@ internal sealed class DurableTaskListener : IListener
             this.functionName = functionName;
             this.functionType = functionType;
             this.connectionName = connectionName;
-#if !FUNCTIONS_V1
 
+#if !FUNCTIONS_V1
             this.scaleMonitor = new Lazy<IScaleMonitor>(() =>
-                this.config.GetScaleMonitor(
+                ScaleUtils.GetScaleMonitor(
+                    this.config.DefaultDurabilityProvider,
                     this.functionId,
                     this.functionName,
-                    this.connectionName));
+                    this.connectionName,
+                    this.config.Options.HubName));
 
 #endif
 #if FUNCTIONS_V3_OR_GREATER
```

Comment:
In SC we have "FUNCTIONS_V3_OR_GREATER", right?

Reply:
I believe so. SC has TFM `.net6.0` and `FUNCTIONS_V3_OR_GREATER` targets `netcoreapp3.1`, which is "older" than `.net6.0`. So yes, I believe this code triggers in SC.